sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/HomeScreen.kt

```diff
@@ -37,13 +39,42 @@ fun HomeScreen(
     val showLoading: IsLoading by viewModel.showLoading.collectAsState()
     val uiModels: List<UiModel> by viewModel.uiModels.collectAsState()
 
+    CheckCameraPermission()
```

Comment:
I could be wrong, but should composable functions be named as a "noun", not a "verb"? 💭 
```suggestion
    CameraPermission()
```

Reply:
@Tuubz Fixed in https://github.com/nimblehq/android-templates/pull/375/commits/108f99bbe905146a8ae4a93da059979976a1ef7a